dogs-initial/src/main/java/com/lambdaschool/dogsinitial/DogsinitialApplication.java

```diff
@@ -7,7 +7,7 @@
 public class DogsinitialApplication
 {
 
-    static DogList ourDogList;
+    public static DogList ourDogList;
     public static void main(String[] args)
     {
         ourDogList = new DogList();
```

Comment:
Another reason why your `NoHandlerFoundException` handling won't work yet: you didn't assign the running Spring application to an `ApplicationContext` and use it for a `DispatcherServlet`. You can do this with the code below:
```
ApplicationContext ctx = SpringApplication.run(DogsinitialApplication.class, args);

DispatcherServlet dispatcherServlet = (DispatcherServlet) ctx.getBean("dispatcherServlet");
dispatcherServlet.setThrowExceptionIfNoHandlerFound(true);
```